src/main/java/com/raisetech/raisetechtask/controller/UserController.java

```diff
@@ -17,22 +21,15 @@ public UserController(UserService userService) {
     }
 
     //ユーザー情報一覧
-    @GetMapping("/")
-    public List<UserResponse> getAll() {
-
-        return userService.findAll().stream().map(UserResponse::new).toList();
-    }
-
     @GetMapping("/{id}")
```

Comment:
`@RequestMapping("/users")`を消したのでこちらも
```suggestion
    @GetMapping("/users/{id}")
```
のようにしたいです！

Reply:
変更後は他の影響箇所も確認しないとでした！

[a61d7d9](https://github.com/HidenoriMurai/raisetech-task9/pull/3/commits/a61d7d9c73ff94fe82f95ca8b9a4f79b736c7b42)